tests/conftest.py

```diff
@@ -120,6 +120,19 @@ def fake_responses():
                 }
             ]
         },
+        ("routegroups.zalando.org/v1", "routegroups"): {
```

Comment:
I think the first tuple entry (API group version) is wrong and does not need the "routegroups"